content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md

```diff
@@ -0,0 +1,263 @@
+
+   | Credential ID | Type                | Where to Use                       |
+   | ------------- | ------------------- | ---------------------------------- |
+   | host          | kubeconfig          | The Host Cluster for testing       |
+   | shire         | kubeconfig          | The Member Cluster for production  |
+   | rohan         | kubeconfig          | The Member Cluster for development |
+   | dockerhub-id  | Account Credentials | Docker Hub                         |
+   | sonar-token   | Secret Text         | SonarQube                          |
+
+   {{< notice note >}}
+
+   You have to manually enter the kubeconfig of your Member Clusters when creating the kubeconfig credentials `shire` and `rohan`. Make sure your Host Cluster can access the APIServer addresses of your Member Clusters.
+
+   {{</ notice >}}
+
+3. You will have five credentials in total.
+
+   ![credentials-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/credentials-created.png)
+
+### Step 6: Create a pipeline
+
+1. Go to the **Pipelines** page and click **Create**. In the dialog that appears, enter `build-and-deploy-application` for **Name** and click **Next**.
+
+   ![pipeline-name](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/pipeline-name.png)
+
+2. In the **Advanced Settings** tab, click **Create** to use the default settings.
+
+3. The pipeline created will display in the list. Click it to go to its detail page.
+
+   ![pipeline-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/pipeline-created.png)
+
+4. Click **Edit Jenkinsfile** and copy and paste the following contents. Make sure you replace the value of `DOCKERHUB_NAMESPACE` with your own value, and then click **OK**.
+
+   ```groovy
+   pipeline {
+     agent {
+       node {
+         label 'maven'
+       }
+   
+     }
+     parameters {
+           string(name:'BRANCH_NAME',defaultValue: 'master',description:'')
+       }
+     environment {
+           DOCKER_CREDENTIAL_ID = 'dockerhub-id'
+           PROD_KUBECONFIG_CREDENTIAL_ID = 'shire'
+           TEST_KUBECONFIG_CREDENTIAL_ID = 'host'
+           DEV_KUBECONFIG_CREDENTIAL_ID = 'rohan'
+   
+           REGISTRY = 'docker.io'
+           DOCKERHUB_NAMESPACE = 'shaowenchen'
+           APP_NAME = 'devops-java-sample'
+           SONAR_CREDENTIAL_ID = 'sonar-token'
+           TAG_NAME = "SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER"
+       }
+     stages {
+       stage('checkout') {
+         steps {
+           container('maven') {
+             git branch: 'master', url: 'https://github.com/kubesphere/devops-java-sample.git'
+           }
+         }
+       }
+       stage('unit test') {
+         steps {
+           container('maven') {
+             sh 'mvn clean -o -gs `pwd`/configuration/settings.xml test'
```

Comment:
I'm wondering if you can really run this script well. Because the option `-o` means offline. Normally, users' environment might do not have the maven cache files.

Reply:
I tested the pipeline and it run well.
<img width="1262" alt="Xnip2021-05-18_11-56-13" src="https://user-images.githubusercontent.com/75110798/118587980-1b0f4c00-b7d0-11eb-8a57-1615de062b00.png">

Reply:
Or, you can highlight the meaning of this option `-o` in case if someone is not familiar with it.

Reply:
Please review the notice I added in lines 225 - 229 [in this commit](https://github.com/kubesphere/website/pull/1576/commits/79cc6e3742add99463e756d2ea7a7644bda6d768).